### DISCUSSION TITLE
Just curious why is Zebar a different app than GlazeWM?

### DISCUSSION THREAD

**Discussion by thesobercoder** [glzr-io/glazewm#712] [Q&A]
Just curious why is Zebar a different app than GlazeWM? Not complaining but, shouldn't this be like a unified experience? Having one config file to control both will be such a great experience, with options to turn off Zebar if one doesn't need it.

**Comment by Razcall** [comment]
I tried you just need to remove the launch command in the configuration if you want to avoid launching zebar

Although completely agree with you the zebar feels sluggish next the the blazingly fast glaze
I suppose the rewrite of glaze in rust was already a heavy well conducted job from the author that he did not yet had the time to rewrite zebar

**Comment by airtonix** [comment]
> Just curious why is Zebar a different app than GlazeWM? Not complaining but, shouldn't this be like a unified experience? Having one config file to control both will be such a great experience, with options to turn off Zebar if one doesn't need it.

no.
